packages/fxa-content-server/app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -13,27 +13,27 @@
 
         {{^isIos}}
             <div class="button-row save-options">
-                <a href="#" class="save-option download-option">
+                <button type="button" href="#" class="save-option download-option">
```

Comment:
We should be able to remove `href="#"` on all of these now.

Also, I don't believe we need `type="button"` unless these guys are in a `form` since users can't submit if they're not in one. Some of what's been changed are in forms and some are not, but that's not a big deal.

Reply:
Good find on the hrefs. Missed those.

Yes, agree - we don't need `type="button"` on buttons not inside a form. I noticed the buttons inside the 2FA modal are inside a form so I added them, and then kinda went to town. I suppose they couldn't hurt, but I've gone ahead and removed the excess ones.